infrastructure/cdn-in-a-box/traffic_monitor/Dockerfile

```diff
@@ -64,7 +63,7 @@ CMD /run.sh
 HEALTHCHECK --interval=10s --timeout=1s \
     CMD bash -c 'source /to-access.sh && [[ "$(curl -s http://trafficmonitor.infra.ciab.test/api/traffic-ops-uri)" == "$TO_URL" ]]'
 
-FROM trafficmonitor-dependencies as get-delve
+FROM trafficmonitor as get-delve
```

Comment:
Seems like this is reverting the changes from #5866 

Same thing in Enroller/Ops etc.

Reply:
Rebased and took out the commits that modify those files.